jsonapi-matchers.gemspec

```
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activesupport", ">= 4.0", "< 6"
+  spec.add_dependency "activesupport", ">= 4.0", "< 6.1"
```

Maybe this is better?
```suggestion
  spec.add_dependency "activesupport", ">= 4.0", "<= 6.0.1"
```

---

Since the goal of this version change is to allow Rails 6.0.x, but limit compatibility below 6.1 since we haven't tested on 6.1 I think we should go with `< 6.1`.